frame/staking/src/pallet/mod.rs

```diff
@@ -774,6 +776,10 @@ pub mod pallet {
 		CommissionTooLow,
 		/// Some bound is not met.
 		BoundNotMet,
+		/// Nominations are not decodable. This means that `Config::MaxNominations` has been
```

Comment:
This _could_ mean what you said, but it could also be for various other reasons as well.

Reply:
Alright, I'll remove this clarification then.

Reply:
It's clarified further down the road